content/en/blog/_posts/2022-05-23-service-ip-dynamic-and-static-allocation.md

```diff
@@ -101,7 +101,7 @@ pie showData
 #### Service IP CIDR block: 10.96.0.0/20
 
 Range Size: 2<sup>12</sup> - 2 = 4094  
-Band Offset: `min(max(16,256/16),256)` = `min(256,256)` = 256  
+Band Offset: `min(max(16,4096/16),256)` = `min(256,256)` = 256  
```

Comment:
Maybe add a space after comma could be better. 😝
```suggestion
Band Offset: `min(max(16, 4096/16), 256)` = `min(256, 256)` = 256  
```

Reply:
There are other two places with this style. I am not sure if I need to change them all to keep it consistent.
https://kubernetes.io/blog/2022/05/25/contextual-logging/

Reply:
I'd recommend adding a space after the comma since earlier in the doc, a similar formatting has been used in line #77. For the two other places with the same style, if it's in this same documentation/otherwise I'd recommend doing it over this PR since a minor space nit might not really warrant a separate PR.

Reply:
Updated.